lib/Listener/FileCreatedFromTemplateListener.php

```diff
@@ -54,18 +54,18 @@ public function handle(Event $event): void {
 			return;
 		}
 
-		if ($this->templateManager->isSupportedTemplateSource($templateFile->getExtension())) {
-			// Only use TemplateSource if supported filetype
-			$this->templateManager->setTemplateSource($event->getTarget()->getId(), $templateFile->getId());
-		}
-
 		if ($this->capabilitiesService->hasFormFilling()) {
 			try {
-				$filledTemplate = $this->templateFieldService->fillFields($templateFile, $event->getTemplateFields());
+				$filledTemplate = $this->templateFieldService->fillFields($templateFile, $event->getTemplateFields(), null, event->getTarget()->getExtension());
```

Comment:
This is missing a `$`
```suggestion
				$filledTemplate = $this->templateFieldService->fillFields($templateFile, $event->getTemplateFields(), null, $event->getTarget()->getExtension());
```